docs/guides/cache/azblob.md

```diff
@@ -0,0 +1,57 @@
+# Azure Blob Storage cache storage
+
+The `azblob` cache store uploads your resulting build cache to
+[Azure's blob storage service](https://azure.microsoft.com/en-us/services/storage/blobs/).
+
+> *Note*
+>
+> The `azblob` cache storage backend requires using a different driver than
+> the default `docker` driver - see more information on selecting a driver
+> [here](../drivers/index.md). To create a new docker-container driver (which
+> can act as a simple drop-in replacement):
+>
+> ```console
+> docker buildx create --use --driver=docker-container
+> ```
+
+To import and export your cache using the `azblob` storage backend we use the
+`--cache-to` and `--cache-from` flags and point it to our desired blob using
+the required `account_url` and `name` parameters:
+
+```console
+$ docker buildx build --push -t <user>/<image> \
+  --cache-to type=azblob,account_url=https://myaccount.blob.core.windows.net,name=my_image \
+  --cache-from type=azblob,account_url=https://myaccount.blob.core.windows.net,name=my_image
+```
+
+## Authentication
+
+To authenticate to Azure to read from and write to the cache, the following
+parameters are required:
+
+* `secret_access_key`: secret access key
+  * specifies the primary or secondary account key for your Azure Blob
+    Storage account. [Azure Blob Storage account keys](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-keys-manage)
+
+While these can be manually provided, if left unspecified, then the credentials
+for Azure will be pulled from the BuildKit server's environment following the
+environment variables scheme for the [Azure Go SDK](https://docs.microsoft.com/en-us/azure/developer/go/azure-sdk-authentication).
+
+> *Warning*
+>
+> These environment variables **must** be specified on the BuildKit server, not
+> the `buildx` client.
+
+## Cache options
+
+The `azblob` cache has lots of parameters to adjust its behavior.
```

Comment:
We can link to buildkit readme section or copy paste options available here. I have a preference to copy content here so users are not confused about buildctl vs. buildx.

Reply:
Yeah, I've tried to summarize the options neatly for the `registry` / `local` / `gha` cache backends, the only reason I haven't put the whole list here is that the `azblob`/`s3` pages are not fully complete yet. I can just paste them in for now, but I'd prefer to do it with some explanation text as well in a follow-up for these?

Reply:
Sure let's see in a follow-up